WarOfEmpires.Domain.Tests/Players/PlayerTests.cs

```diff
@@ -948,5 +948,72 @@ public sealed class PlayerTests {
 
             player.SiegeWeapons.Single().Count.Should().Be(2);
         }
+
+        [TestMethod]
+        public void Player_HealTroops_Succeeds() {
+            var player = new Player(0, "Test");
+            typeof(Player).GetProperty(nameof(Player.Resources)).SetValue(player, new Resources(1000000, 100000, 100000, 100000, 100000));
+            typeof(Player).GetProperty(nameof(Player.Stamina)).SetValue(player, 90);
+            player.TrainTroops(30, 10, 20, 10, 20, 10);
+            var previousResources = player.Resources;
+            player.HealTroops(10);
+            player.Stamina.Should().Be(100);
+            player.Resources.Should().Be(previousResources - new Resources(food: 2000));
+        }
+
+        [TestMethod]
+        public void Player_HealTroops_Succeeds_ForZeroTroops() {
```

Comment:
A nitpick: I try to put all words in test names with _ as spaces except the ones that are a class or function name.

Reply:
Renamed Player_HealTroops_Succeeds_For_Zero_Troops()